Move mov_fn agg to module (#90836)
elastic/elasticsearch: modules/aggregations/src/main/resources/org/elasticsearch/aggregations/moving_function_whitelist.txt

# Side Public License, v 1.
#

> # This file contains a whitelist for the Moving Function pipeline aggregator

Reviewer: ```suggestion
# This file contains an allow-list for the Moving Function pipeline aggregator
```
